module-1/src/Components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,78 @@
+import { ReactNode } from "react";
+import { createPortal } from "react-dom";
+import "./Dialog.css";
+const mountElement: any = document.getElementById("overlays") || <div />;
+
+interface IDialogProps {
+  openDialog: boolean;
+  dialogTitle?: string;
+  children?: ReactNode;
+  closeCallback: () => void;
+  submitCallback: () => void;
+}
+
+export const Dialog = ({
+  openDialog = false,
+  dialogTitle,
+  children,
+  closeCallback,
+  submitCallback,
+}: IDialogProps) => {
+  if (!openDialog) return null;
+  function handleClose(): void {
```

Comment:
Too obvious and doesn't require full typing.
```suggestion
  function handleClose() {
```

Reply:
updated